base/distributed/cluster.jl

```diff
@@ -1036,7 +1036,7 @@ function disable_nagle(sock)
     @static if Sys.islinux()
         # tcp_quickack is a linux only option
         if ccall(:jl_tcp_quickack, Cint, (Ptr{Void}, Cint), sock.handle, 1) < 0
-            warn_once("Networking unoptimized ( Error enabling TCP_QUICKACK : ", Libc.strerror(Libc.errno()), " )")
+            @warn "Networking unoptimized ( Error enabling TCP_QUICKACK : $(Libc.strerror(Libc.errno())) )"
```

Comment:
Missing `max_log=1`?

Reply:
Good catch.  You can tell this is one i hand edited :)